com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -221,6 +221,12 @@ void SetPerCameraShaderVariables(CommandBuffer cmd, ref CameraData cameraData)
 
             cmd.SetGlobalVector(ShaderPropertyId.screenSize, new Vector4(cameraWidth, cameraHeight, 1.0f / cameraWidth, 1.0f / cameraHeight));
 
+            // Calculate a bias value which corrects the mip lod selection logic when image scaling is active.
+            // We clamp this value to 0.0 or less to make sure we don't end up reducing image detail in the downsampling case.
+            float mipBias = Math.Min((float)-Math.Log(cameraWidth / scaledCameraWidth, 2.0f), 0.0f);
+            cmd.SetGlobalFloat(ShaderPropertyId.globalMipBias, mipBias);
+            cmd.SetGlobalFloat(ShaderPropertyId.globalMipBiasPow2, Mathf.Pow(2.0f, mipBias));
```

Comment:
I'm curious about if it's worth to bake this into a single vector property. There's still some mid range mobile GPUs out there that are vector based. On the other hand everything modern is scalar based and I guess making vector adds more VGPR.

Reply:
I don't think it'll hurt things on modern GPUs so I just updated the diff to combine the constants into a single vector.